### DISCUSSION TITLE
Docker FMU platforms in OMEdit

### DISCUSSION THREAD

**Discussion by AnHeuermann** [OpenModelica/OpenModelica#8483] [General]
### Description

Follow-up on #8168.

Currently the OMEdit FMU platform settings are looking like this:
![image](https://user-images.githubusercontent.com/38031952/144415986-4bdcabed-30c6-451f-9d39-0728e3d59118.png)

In my opinion this dialog has a number of issues:

1. The names are confusing.
    - It's not directly what command corresponds to which FMU platform (`win64`, `win32`, `linux64`, `linux32`).
    - The names are long and hard to read.
2. They use internal docker argument the user don't know.
    - `--pull=never` is only there so that OMEdit doesn't freeze while pulling a large image (several GB). Users need to do that in a different shell or add the command to additional platforms.
3. What is a valid option for `additional platforms`?

What are your opinions about this?

----------------------------

My proposal for 1 and 2:
Order the platforms after the available FMU platforms (`win64`, `win32`, `linux64`, `linux32`) and make it obvious that you can only choose max one target for each FMU platform.
Use a short name like `x86_64-linux via docker` (and so on) as name and add a tool tip displaying the full command when moving the mouse over it.

My proposal for 3:
Add a link to the User's guide section about this directly in OMEdit.

**Comment by sjoelund** [comment]
> Order the platforms after the available FMU platforms (win64, win32, linux64, linux32) and make it obvious that you can only choose max one target for each FMU platform.

The problem is we don't know which platform is used until we run the configure-script in the image...

**Comment by sjoelund** [comment]
> Use a short name like x86_64-linux via docker (and so on) as name and add a tool tip displaying the full command when moving the mouse over it.

The problem with that is you probably want to copy-paste the actual line, and you cannot do that with tooltips.
